docs/docs/40-contributor-guide/10-hacking-on-kargo.md

```diff
@@ -455,13 +455,11 @@ make hack-serve-docs
 ```
 
 :::info
-If you wish to opt-out of executing code-generation within a container (for
-performance reasons, perhaps), drop the `hack-` prefix from the target to run the docs natively on your system:
+If you want to build and serve the docs on your local machine, run the following command:
```

Comment:
I wish we wouldn't change this. The contributor docs were deliberately written to encourage using container-based options because it reduces the incidence of problems arising from the contributor's local environment. As it was worded before, it was less "here's how to do it because you want to," and more, "here's how to do it if you must."

Reply:
we can add a line stating that this is not something that we recommend doing instead. WDYT?